events/events-team/best-practices.md

```diff
@@ -40,26 +58,27 @@ Accessibility
 
 If the event is being held during KubeCon/CloudNativeCon it's important to get a space that is not with other co-located events to avoid confusion like we have had in the past.
 
-## Social
-Hallway Track is listed as the favorite track at every event. The social is a longer hallway track that doesn't interfere with content. The social should have fun activities!  
+## Social / Celebration
+The Hallway Track is listed as the favorite track at every event. The social/celebration is a longer hallway track that doesn't interfere with content. The social should have fun activities!  
 
 ## Communication and Advertising  
-kubernetes-dev@googlegroups.com is our artery for contributor communication and should be the main news network for contributor events.  
+[kubernetes-dev@googlegroups.com][k-dev] is our artery for contributor communication and should be the main news network for contributor events.  
 Other channels:  
-Slack - #contributor-summit, #kubecon, #kubernetes-dev  
-Twitter  
-Kubernetes blog
-discuss.kubernetes.io
-Thursday Community Meetings
+Slack - [#contributor-summit], [#kubecon], [#kubernetes-dev]  
+[Twitter]  
+[Kubernetes blog]  
+[discuss.kubernetes.io]  
+[Weekly Thursday Community Meetings][community-meetings]
 
 ## Sponsorship
 We like to keep our events clear from sponsorships as it distracts from the mission.
 
 ## Swag / T-Shirts
 What messaging do we want to convey with the swag?  
 thockin has designed all contributor summit NA shirts and jberkus for Shanghai; both have done an EU shirt  
+
 Best practices:  
-Order shirts 10 days before the event to make sure everyone gets the size that they want unless there are custom/shipping issues.  
+Order shirts well in advance before the event to make sure everyone gets the size that they want in case there are custom/shipping issues.  
```

Comment:
10 days before the event will ensure that everyone gets the right size because we should have all sizes by then

Reply:
Since the shirts are ordered through CNCF I think it would make sense to have them set the deadline for that, instead of us having that defined here, do you agree?

Reply:
agree but we haven't taken this advice in the past and have been burned 3 times or have many extras left and nothing to do with them. figured a best practices doc would be good for all of this and not just cncf related vs us.

Reply:
I don't want to argue logistics on ordering and shipping hundreds of t-shirts, I was simply recommending "well in advance" as a general rule, instead of having a hard "10 days before". I'm fine changing this back if you want.

Reply:
we can leave the edit as is. merging now.